sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -400,8 +402,8 @@
      * @throws HttpResponseException If the {@code} key is an empty string.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<ConfigurationSetting> getSetting(ConfigurationSetting setting) {
-        return withContext(context -> getSetting(setting, context))
+    public Mono<ConfigurationSetting> getSetting(String key, String label, String dateTime) {
```

Comment:
DateTime as a string? I would be hoping for an Instant. Having it as a string brings up many questions... like "What format should it be? What locale do I parse this string in?"

Reply:
Agreed, have this take an `OffsetDateTime` and we will handle the string conversion